apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/sources/ContractSourcesPage.tsx

```diff
@@ -91,7 +94,7 @@ const VerifyContractModal: React.FC<
         <ModalHeader>
           <Flex gap={2} align="center">
             <Heading size="subtitle.md">Contract Verification</Heading>
-            <Badge variant="outline" colorScheme="purple" rounded="md" px={2}>
+            <Badge variant="outline" className="px-2">
```

Comment:
keep the default padding

Reply:
Badge doesn't have primary variant tho, only `"outline" | "default" | "destructive" | "secondary" | "success" | "warning"`

Reply:
yeah - the default is primary

Reply:
updated!